client/ayon_silhouette/api/pipeline.py

```diff
@@ -148,6 +159,34 @@ def _set_current_context_label(action):
         action.setToolTip("Set active session resolution")
         action.triggered.connect(_on_set_resolution)
 
+        menu.addSeparator()
+
+        # region Workfile templates
+        menu_template = menu.addMenu("Template Builder")
+
+        action = menu_template.addAction("Build Workfile from template")
+        action.triggered.connect(lambda: build_workfile_template())
+
+        menu_template.addSeparator()
+
+        action = menu_template.addAction("Open template")
+
+        def _open_template_ui():
```

Comment:
Shouldn't this function be moved to "global scope" so it is not garbage collected?

Reply:
It works fine as it is - but I can move it. It's basically like a lambda - but readable for this length :P